proxies/live/apiproxy/policies/AssignMessage.AddCors.xml

```diff
@@ -4,7 +4,7 @@
     <Set>
         <Headers>
             <Header name="Access-Control-Allow-Origin">{request.header.origin}</Header>
-            <Header name="Access-Control-Allow-Headers">origin, x-requested-with, accept, content-type, nhsd-session-urid</Header>
+            <Header name="Access-Control-Allow-Headers">origin, x-requested-with, accept, content-type, if-match, nhsd-session-urid</Header>
```

Comment:
Why have we added "if-match"?

Reply:
Suspect it would be the result of a couple of merges, will remove